eth_abi/utils/padding.py

```diff
@@ -1,15 +1,33 @@
-import functools
+def just(value, length, fillerchar, direction='left'):
+    if isinstance(value, str):
+        _fillerchar = fillerchar.decode()
```

Comment:
I think it's better to just fix the calls that are using a fillerchar of the wrong type.